Makefile

```diff
@@ -0,0 +1,33 @@
+# Minimal makefile for Sphinx documentation
+# You can set these variables from the command line, and also
+# from the environment .
+
+LANGUAGES     ?= en de zh_CN es fr nl ja
+SPHINXINTL    ?= sphinx-intl
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = source
+BUILDDIR      = build
+LOCALEDIR     ?= locale
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+html: $(addprefix html/,$(LANGUAGES)) $(BUILDDIR)/html/index.html $(BUILDDIR)/html/CNAME
+	@echo "Pages available at file://$$PWD/$(BUILDDIR)/html/index.html"
+
+$(addprefix html/,$(LANGUAGES)): $(MAKEFILES)
+	@$(SPHINXBUILD) "$(SOURCEDIR)" "$(BUILDDIR)/$@" $(SPHINXOPTS) -Dlanguage=$(word 2,$(subst /, ,$@))
```

Comment:
```suggestion
	@$(SPHINXBUILD) -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)/$@" $(SPHINXOPTS) -Dlanguage=$(word 2,$(subst /, ,$@))
```

Reply:
Sir , I have implemented the dirhtml at : https://henilpanchal.pythonanywhere.com/static/, but on a large number of pages the static assets are not being rendered .we would also have to **change the directory structures** for various sections like learn and also the **navigation of pages** while the pages for the site like https://henilp105.github.io/webpage/en/index can be used. but it can surely be implemented.

Thanks and Regards,
Henil Shalin Panchal